crates/oxc_linter/src/rules/eslint/default_param_last.rs

```diff
@@ -17,26 +17,62 @@ pub struct DefaultParamLast;
 declare_oxc_lint!(
     /// ### What it does
     ///
-    /// Enforce default parameters to be last
+    /// Requires default parameters in functions to be the last ones.
     ///
     /// ### Why is this bad?
     ///
-    /// Putting default parameter at last allows function calls to omit optional tail arguments.
+    /// Placing default parameters last allows function calls to omit optional trailing arguments,
+    /// which improves readability and consistency. This rule applies equally to JavaScript and
+    /// TypeScript functions.
+    ///
+    /// ### Options
+    ///
+    /// No options available for this rule
     ///
     /// ### Examples
     ///
     /// Examples of **incorrect** code for this rule:
-    /// ```javascript
-    /// // Incorrect: optional argument can **not** be omitted
+    /// ```js
+    /// /* default-param-last: "error" */
+    ///
+    /// /* ✘ Bad: */
```

Comment:
```suggestion
```

This is redundant, as we already have `Examples of **incorrect** code for this rule:` above